edx_exams/apps/lti/views.py

```diff
@@ -235,12 +235,12 @@ def start_proctoring(request, attempt_id):
     lti_config = LtiConfiguration.objects.get(id=lti_config_id)
 
     proctoring_start_assessment_url = urljoin(
-        get_lti_root(),
+        get_lti_api_base(),
```

Comment:
this helper function already existed in the lti library. Removed a 'duplicate' util from this service